extensions/machine-learning/package.json

```diff
@@ -7,7 +7,7 @@
   "preview": true,
```

Comment:
you changed the version in line #5 as 0.2.2 right (somehow shows 0.2.1 if I expand here)

Reply:
/headscratch

https://github.com/microsoft/azuredatastudio/blob/main/extensions/machine-learning/package.json#L5

Yeah - weird. Github issue I guess.